packages/mdx/mdx-hast-to-jsx.js

```diff
@@ -105,10 +105,10 @@ function toJSX(node, parentNode = {}, options = {}) {
       } class MDXContent extends React.Component {
   constructor(props) {
     super(props)
-    this.layout = ${layout}
+    this.layout = ${layout || 'null'}
   }
   render() {
-    const { components = {} } = this.props
+    const { components = {}, ...props } = this.props
```

Comment:
won't this need an equivalent change in the runtime to pass the props into createElement?

Previously, the runtime was just using the scoped names when passed in to defined the `components` and `props` variables. If we're standardizing the class to be `this.props` access always, then we need to pass everything into createElement in the runtime like

```js
createElement(MDXContent, {components, ...props}, children)
```